tests/codegen/waiter-tests/src/test/kotlin/com/test/FunctionTypeTest.kt

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.test
+
+import com.test.model.*
+import com.test.utils.successTest
+import com.test.waiters.*
+import org.junit.jupiter.api.Test
```

Comment:
Use `kotlin.test.Test` instead, it's better for KMP compatibility (even if we only plan to run codegen on JVM)

Reply:
Now that I notice that I see I have to do the same for the rest of the [tests](https://github.com/awslabs/smithy-kotlin/tree/3934d48a9e084e23b1250501b0deb8dafe2a1c03/tests/codegen/waiter-tests/src/test/kotlin/com/test)